updater/lib/dependabot/dependency_change.rb

```diff
@@ -122,6 +138,27 @@ def merge_changes!(dependency_changes)
       updated_dependency_files.compact!
     end
 
+    sig { returns(T::Boolean) }
+    def previous_version?
+      return true if updated_dependencies.all?(&:requirements_changed?)
+      return true if updated_dependencies.all?(&:previous_version)
+
+      false
+    end
+
+    sig { void }
+    def check_dependencies_have_previous_version
+      return if updated_dependencies.all?(&:requirements_changed?)
```

Comment:
wondering if this is already covered by the `previous_version?` method above?

return if previous_version?